tests/units/DatabaseUpgradeTest.php

```diff
@@ -114,3 +145,36 @@ public function isMigrated(): bool
         return true;
     }
 }
+
+
+class PriorityEmergency implements Priority, MigratesTable
+{
+    public function isMigrated(): bool
+    {
+        return false;
+    }
+
+    public function priority(): int
+    {
+        return 0;
+    }
+
+    public function migrateTable(): void
+    {
+        DatabaseUpgradeTest::$calls[] = static::class;
+    }
+}
+
+
+class PriorityNormal implements MigratesTable
```

Comment:
this could be called PriorityDefault or even better, `PriorityImplicit`, since default could mean that there's a default set somewhere within the class, which is not even the case